### PR TITLE
[FEATURE] User Section Created

### PR DESCRIPTION
**Pull request by Oblivious19**
### **Description**
User Section Created

- [x] Added Paddings

- [x] Added Margins where required

- [x] Added Responsiveness

FEATURES

Desktop view
image

Tablet View
image

Mobile First 
image

### CODE REVIEW

File: src/components/UserSection/UserSection.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+function UserSection()
+{
+    return <div>  <div class="py-24 bg-gray-50 ">
```
Comment (kamalika0363) [inline src/components/UserSection/UserSection.jsx:4]:
```suggestion
    return <div>  <div class="py-20 px-12 sm:p-32 bg-gray-50 ">
```

File: src/components/UserSection/UserSection.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    <div class="flex-col text-center ">
+   <div class="flex justify-center"> <img  src="/UserSection/Logomark.svg"></img>
```
Comment (kamalika0363) [inline src/components/UserSection/UserSection.jsx:6]:
```suggestion
   <div class="flex justify-center gap-3"> <img  src="/UserSection/Logomark.svg"></img>
```

File: src/components/UserSection/UserSection.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    <img src="/UserSection/Logotype.svg"></img></div>
+    
+    <div class="font-inter text-5xl font-medium px-1 ">
```
Comment (kamalika0363) [inline src/components/UserSection/UserSection.jsx:9]:
```suggestion
    <div class="font-inter text-2xl md:text-3xl lg:text-4xl font-medium px-1 ">
```

File: src/components/UserSection/UserSection.jsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+      
+       <div class="Main my-4"> We’ve been using Untitled to kick start every new project and can’t imagine working without it.</div>
+    </div>
+   
+    <div >
+       <div class=" flex justify-center "> <img  src="/UserSection/Avatar.png"></img></div>
```
Comment (kamalika0363) [inline src/components/UserSection/UserSection.jsx:15]:
```suggestion
       <div class=" flex justify-center mt-8"> <img  src="/UserSection/Avatar.png"></img></div>
```

**Comment by Oblivious19** [comment]
### **Ordered Changes were made**

Before
image

After
image

**Comment by Oblivious19** [comment]
Changed Done

**Review by kamalika0363 — APPROVED** [review]

**Review by kamalika0363 — APPROVED** [review]
LGTM!!

**Review by kamalika0363 — APPROVED** [review]

**Pull request closed without merging**